back/app/models/application_record.rb

```diff
@@ -15,4 +15,11 @@ def self.polymorphic_associations name, as
       claz.reflect_on_all_associations.select{|asc| asc.name == name.to_sym && asc.options[:as] == as.to_sym}.present?
     end
   end
+
+  def self.disable_inheritance(&blk)
```

Comment:
What is this doing exactly? It's preventing Rails from trying to load files that are missing for the core edition?

Reply:
The issue is that for Single Table Inheritance like notifications or email campaigns, when you try to load notifications and the type does not exist, you get errors like this one:
```
ActiveRecord::SubclassNotFound (The single-table inheritance mechanism failed to locate the subclass: 'Notifications::IdeaAssignedToYou'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite Notification.inheritance_column to use another column for that information.)
```

This prevents rails from trying to load the subclasses